fhirAssembly/src/test/java/io/vantiq/extsrc/fhirAssembly/TestFhirAssembly.java

```diff
@@ -710,23 +722,274 @@ public void test310UpdateEtag() throws Exception {
     }
     
     @Test
-    public void test400Delete() throws Exception {
```

Comment:
This is (on my screen here) just a bad Git diff.  400Delete didn't change, but some other tests were added before it.  Tried to keep the tests organized in the same order they run (there are some interdependent) just so things are easier to find.